.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: python
+os:
+  - osx
+
 sudo: false
```

Comment:
Same thing here, let's add something like

```
branches:
 only:
  - dev
```

Reply:
Also the following can be used to integrate tox with travis ... https://pypi.python.org/pypi/tox-travis